src/style.css

```
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+html {
+  font-weight: 300;
+  font-family: Avenir, Helvetica, sans-serif;
+}
+
+.bar {
+  font-weight: 900;
+  font-family: inherit;
+  display: block;
+  width: 100%;
+  box-shadow: 0 1px 8px #3d4e611a;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+}
+
+.bar__big {

```

Modifier would be better 
```suggestion
.bar--big {

```